language/Spanish_Chile/strings.po

```diff
@@ -0,0 +1,6575 @@
+# Spanish Chile translations for MAME package
+# Copyright (C) 1997-2023 MAMEdev and contributors
+# This file is distributed under the same license as the MAME package.
+# Automatically generated, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: MAME\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-02-18 05:29+1100\n"
+"PO-Revision-Date: 2023-02-20 01:53-0300\n"
+"Last-Translator: MELERIX\n"
+"Language-Team: \n"
+"Language: es_CL\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#: src/frontend/mame/ui/ui.cpp:474 src/frontend/mame/ui/ui.cpp:547
+msgid ""
+"\n"
+"\n"
+"Press any key to continue"
+msgstr ""
+"\n"
+"\n"
+"Presiona cualquier tecla para continuar"
+
+#: src/frontend/mame/ui/ui.cpp:559
+msgid ""
+"This system requires media images to be mounted for the following device(s): "
+msgstr ""
+"Este sistema requiere que se monten imágenes de disco para el/los "
+"siguiente(s) dispositivo(s): "
+
+#: src/frontend/mame/ui/ui.cpp:1275
+#, c-format
+msgid ""
+"UI controls enabled\n"
+"Use %1$s to toggle"
+msgstr ""
+"Controles de UI habilitados\n"
+"Usa %1$s para alternar"
+
+#: src/frontend/mame/ui/ui.cpp:1277
+#, c-format
+msgid ""
+"UI controls disabled\n"
+"Use %1$s to toggle"
+msgstr ""
+"Controles de UI deshabilitados\n"
+"Usa %1$s para alternar"
+
+#: src/frontend/mame/ui/ui.cpp:1486
+msgid "Master Volume"
+msgstr "Volumen principal"
+
+#: src/frontend/mame/ui/ui.cpp:1495
+#, c-format
+msgid "%1$s Volume"
+msgstr "Volumen %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1517
+#, c-format
+msgid "Overclock CPU %1$s"
+msgstr "Overclock de CPU %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1525
+#, c-format
+msgid "Overclock %1$s sound"
+msgstr "Overclock de sonido %1$s"
+
+#: src/frontend/mame/ui/ui.cpp:1544
+#, c-format
+msgid "%1$s Refresh Rate"
+msgstr "Tasa de refresco %1$s"
```

Comment:
"Frequencia de actualización" - https://www.intel.es/content/www/es/es/gaming/resources/highest-refresh-rate-gaming.html

Reply:
Literal translations are not good translations.  You need to use correct terminology in the target language.